yamls/jp/code.yaml

```diff
@@ -246,27 +251,31 @@
 
-      - { start: 0x73F4D0, type: bss, vram: 0x8011B830, name: code/8011B830 }
+
+
+      - { start: 0x73F4D0, type:  bss, vram: 0x8011B830, name: code/8011B830 }
       - { start: 0x73F4D0, type: .bss, vram: 0x8011B850, name: code/m_actor }
       - { start: 0x73F4D0, type: .bss, vram: 0x8011B890, name: code/m_actor_dlftbls }
-      - { start: 0x73F4D0, type: bss, vram: 0x8011B8B0, name: code/8011B8B0 }
+      - { start: 0x73F4D0, type:  bss, vram: 0x8011B8B0, name: code/8011B8B0 }
       - { start: 0x73F4D0, type: .bss, vram: 0x80126E70, name: code/m_collision_obj }
-      - { start: 0x74AB40, type: .bss, vram: 0x80126EA0, name: code/m_common_data }
-      - { start: 0x75B600, type: bss, vram: 0x80137960, name: code/80137960 }
+      - { start: 0x73F4D0, type: .bss, vram: 0x80126EA0, name: code/m_common_data }
+      - { start: 0x73F4D0, type:  bss, vram: 0x80137960, name: code/80137960 }
       - { start: 0x73F4D0, type: .bss, vram: 0x80138E50, name: code/m_debug}
-      - { start: 0x73F4D0, type: bss, vram: 0x80138E60, name: code/m_debug_display}
+      - { start: 0x73F4D0, type:  bss, vram: 0x80138E60, name: code/m_debug_display}
       - { start: 0x73F4D0, type: .bss, vram: 0x80141E10, name: code/m_lights }
-      - { start: 0x75B600, type: .bss, vram: 0x80141FA0, name: code/m_malloc }
-      - { start: 0x75B600, type: bss, vram: 0x80141FE0, name: code/80141FE0 }
+      - { start: 0x73F4D0, type: .bss, vram: 0x80141FA0, name: code/m_malloc }
+      - { start: 0x73F4D0, type:  bss, vram: 0x80141FE0, name: code/80141FE0 }
+      - { start: 0x73F4D0, type: .bss, vram: 0x80142C00, name: code/m_npc }
+      - { start: 0x73F4D0, type: .bss, vram: 0x801438B0, name: code/m_npc_walk }
       - { start: 0x73F4D0, type: .bss, vram: 0x80143900, name: code/m_player_call }
       - { start: 0x73F4D0, type: bss, vram: 0x80143910, name: code/80143910 }
       - { start: 0x73F4D0, type: .bss, vram: 0x80144670, name: code/m_submenu }
       - { start: 0x73F4D0, type: bss, vram: 0x80144690, name: code/80144690 }
       - { start: 0x73F4D0, type: .bss, vram: 0x80145020, name: code/game }
-      - { start: 0x73F4D0, type: bss, vram: 0x80145060, name: code/80145060 }
+      - { start: 0x73F4D0, type:  bss, vram: 0x80145060, name: code/80145060 }
       - { start: 0x73F4D0, type: .bss, vram: 0x80145390, name: code/irqmgr }
-      - { start: 0x73F4D0, type: bss, vram: 0x80145620, name: code/80145620}
+      - { start: 0x73F4D0, type:  bss, vram: 0x80145620, name: code/80145620}
       - { start: 0x73F4D0, type: .bss, vram: 0x801462B0, name: code/sys_matrix }
-      - { start: 0x73F4D0, type: bss, vram: 0x801462C0, name: code/801462C0 }
+      - { start: 0x73F4D0, type:  bss, vram: 0x801462C0, name: code/801462C0 }
       - { start: 0x73F4D0, type: .bss, vram: 0x801492C0, name: code/zurumode }
-      - { start: 0x73F4D0, type: bss, vram: 0x801492D0, name: code/801492D0 }
+      - { start: 0x73F4D0, type:  bss, vram: 0x801492D0, name: code/801492D0 }
```

Comment:
What toher entries? The bss is just a block on its own, since it needs to be kept ordered this helps to see the addresses, otherwise it's hard to compare them.